src/parser.js

```diff
@@ -97,15 +97,15 @@ export default class Parser {
       const resource = new Resource();
       resource.href = this.parseURL(section.resources[0].url);
 
-      for (const action of section.resources) {
+      section.resources.forEach((action) => {
```

Comment:
Any reason for this change? Going from something supported in ES6 to something non-standard?